docs/src/main/tut/index.md

```diff
@@ -4,6 +4,10 @@ menu: main
 weight: 1
 ---
 
+**Note**: To run examples, please make sure that the flag `-Ypartial-unification`  
```

Comment:
Should we also note that it's enabled by default by the g8 template?

Reply:
Sure, I'm alright with this. I'm all for anything that makes it more obvious that it's enabled, as it's pretty much the most common source of errors currently.

Reply:
updated.